HunterJuneau/dine-and-dash
# Bj detailed order view

## Description
This view shows the product orders and all details for an order. 

## How Can This Be Tested?
To test this feature, head to a user's profile. View their orders, then click the 'see order details' button. 

## Types of changes
